nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/pubsub/AbstractGCPubSubProcessor.java

```diff
@@ -58,6 +73,19 @@ public abstract class AbstractGCPubSubProcessor extends AbstractGCPProcessor imp
 
     private static final Set<Relationship> relationships = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(REL_SUCCESS, REL_FAILURE)));
 
+    @Override
+    public List<PropertyDescriptor> getSupportedPropertyDescriptors() {

```

Comment:
This change also impacts `ConsumeGCPubSubLite` and `PublishGCPubSubLite`, so those processors should be adjusted to leverage these properties.

Reply:
I can't find how to configure proxy in PubSub Lite. Do you have any pointer ?

Reply:
It looks like the `PartitionAssignmentServiceSettings` and `PartitionAssignmentServiceClient` on the `SubscriberSettings` builder provide a way to customized the transport channel with proxy settings. On the other hand, making changes to the PubSub Lite changes could be handled separately if you introduced a new intermediate abstract class that supported proxy credentials for just the standard Publish and Consume processors.

Reply:
I chose the second option. I'm not confident if I can change the pubsub lite processor.  Maybe later I'll try it but I'll do it on another PR/Jira.